course/auth.py

```diff
@@ -119,15 +120,25 @@ def __init__(self, impersonator, *args, **kwargs):
 
         self.fields["user"] = forms.ChoiceField(
                 choices=[
-                    (u.id, "%s - %s, %s" % (u.email, u.last_name, u.first_name))
-                    for u in sorted(impersonees,
-                        key=lambda user: user.last_name.lower())
-                    ],
+                    # Translators: information displayed when select user
+                    # for impersonating, you can use your prefered
+                    # style. For example %(user_lastname)s,
+                    # %(user_firstname)s (user_email)s, but all three
+                    # strings should be used.
+                    (u.id, _("%(user_email)s - %(user_lastname)s, "
```

Comment:
In fact, I add that statement mainly for Chinese users, for convenience, as we always use full name with the format %(lastname)s%(firstname)s, without separator between the two string